www/templates/intro/login.html

```diff
@@ -1,11 +1,11 @@
 <ion-content>
 <div class="intro-title">
-<center><h3>Login via google</h3></center></div>
+<center><h3>Login using a google account</h3></center></div>
 
 <div class="intro-text">
 Currently, we only support logging in via google, since they support techniques
-such as two factor authentication for greater security. Participants at UC
-Berkeley can choose to login using either their CalNet ID or a personal gmail
+such as two factor authentication for greater security. Participants at 
+IISc can choose to login using either their iiscwlan or a personal gmail
```

Comment:
participants at IISc can't login using their iiscwlan; you have not yet integrated with the iisc authentication system, whatever it is. They can only login using personal gmail addresses.